doc/user/user/samples/index.md

```diff
@@ -31,21 +31,25 @@ The samples listing shows high-level sample details, such as:
 
 ### Viewing individual sample details
 
+**Note: Details for individual samples can be viewed from anywhere a sample is listed (such as from the recent activity, analysis results, analysis output files, sharing of samples, etc) by clicking on the sample name. This will launch the sample details viewer. Only a user with the project Manager role will be able to modify sample details, metadata, and files.**
+
+![Sample details page.]({{ site.baseurl }}/images/tutorials/common/samples/sample-detail-viewer.png)
```

Comment:
I feel this image is out of place here.  You are showing the user the contents of the viewer without initially showing them how to open it, which is the subsequent image

Reply:
After discussing with @deepsidhu85 this will be updating in a future documentation upgrade.